charts/kube-prometheus-stack/templates/thanos-ruler/ruler.yaml

```diff
@@ -31,10 +31,12 @@ spec:
   listenLocal: {{ .Values.thanosRuler.thanosRulerSpec.listenLocal }}
   serviceAccountName: {{ template "kube-prometheus-stack.thanosRuler.serviceAccountName" . }}
 {{- if .Values.thanosRuler.thanosRulerSpec.externalPrefix }}
-  externalPrefix: "{{ tpl .Values.thanosRuler.thanosRulerSpec.externalPrefix . }}"
+  externalPrefix: {{ tpl .Values.thanosRuler.thanosRulerSpec.externalPrefix . | quote }}
```

Comment:
@jkroepke coming back to reviewing those, is that still needed with the PR to clean up the `UsesHelmValues ` ?